di_website/spotlight/models.py

```diff
@@ -125,3 +157,30 @@ class Meta():
     ]
 
     search_fields = [index.SearchField('ddw_id'), index.SearchField('title')]
+
+
+class CountrySpotlight(TypesetBodyMixin, HeroMixin, Page):
+    country_spotlight = StreamField(
+        StreamBlock([
+        ('add_spotlight_page', PageChooserBlock(required=False, target_model='spotlight.SpotlightPage')),
+    ],
+        blank=True,
+        max_num=2,
+        block_counts={
+            'add_spotlight_page': {'max_num': 2},
+        }
```

Comment:
Nooo... I'm optimistic that we may have more spotlights than these ... wouldn't want to have to make a code change just to accommodate that

Reply:
Nice one though ... `block_counts` ... one of the good things of doing reviews is that u learn

Reply:
We can now add unlimited spotlight pages to the country spotlights